CONTRIBUTING.md

```diff
@@ -46,7 +46,9 @@ The more information an issue includes, the better! For example:
 - If you have the rights: set yourself as an assignee and add the __in progress__ label.
 - [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the Klaw repository.
 - On your fork, [create a branch](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/making-changes-in-a-branch/managing-branches#creating-a-branch) named after the issue you're working on.
-- Make all of your changes 🧑‍💻
+- 🧑‍💻 Make all of your changes. You may find detailed documentation about how to set up a local development environment in our documentation:
+  - [for `coral` development](coral/docs/README.md)
```

Comment:
I would add a link to `coral/README.md#installation-and-usage` 

There's an overview over the different ways to setup an environment with a short explanation and a "whey to use it" etc. I think that's the better place to check that out if people are starting new.